src/payment-information/payment-information.service.ts

```diff
@@ -0,0 +1,149 @@
+import { Injectable, Logger, NotFoundException } from '@nestjs/common';
+import { CreateCCInformationInput } from './dto/create-creadit-card-information.input';
+import { InjectModel } from '@nestjs/mongoose';
+import { PaymentInformation } from './entities/payment-information.entity';
+import { Model } from 'mongoose';
+import { PaymentMethod } from 'src/payment-method/payment-method.enum';
+import { User } from 'src/graphql-types/user.entity';
+import { UserPaymentInformation } from './entities/user-payment-information.entity';
+
+@Injectable()
+export class PaymentInformationService {
```

Comment:
what about the other payment methods besides credit card? did I miss the code or is this planned for a future PR?

Reply:
I dont think it is necessary to add the graphql mutation, since they will be automatically added.